### ISSUE TITLE
Focus on Next and Previous buttons

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#69]
See https://github.com/Yoast/wordpress-seo/issues/5528
On the Onboarding Wizard, when using only the keyboard and pressing the "Next" and "Previous" buttons, focus stays on the buttons. This is not ideal for accessibility for the reasons mentioned in https://github.com/Yoast/wordpress-seo/issues/5528

It seems to me the buttons are not completely re-rendered on each step, while their properties yes, they get updated. This is good for a11y because screen readers may get confused then elements are re-rendered. On the other hand, the buttons have focus and even though the step is updated, the focus stays on the button. This is native browsers behaviour :)

Ideally, when pressing "Next" and "Previous" the focus should be set  on a proper element at the top of the wizard.
